sections/04_study.md

```diff
@@ -241,114 +241,62 @@ this was easy to simulate with a computational model. As we learn to better
+But the reliable ground truth provided by these assays makes promoter identification an appealing test bed for deep learning approaches that can also identify enhancers.
+
+#### Enhancers
+
+Recognizing enhancers presents additional challenges.
+Enhancers may be up to 1,000,000 bp away from the affected promoter, and even within introns of other genes [@doi:10.1038/nrg3458].
+Enhancers do not necessarily operate on the nearest gene and may affect multiple genes.
+Their activity is frequently tissue- or context-specific.
+No biochemical assay can reliably identify all enhancers.
+Distinguishing them from other regulatory elements remains difficult, and some believe the distinction somewhat artificial [@doi:10.1016/j.tig.2015.05.007].
+While these factors make the enhancer identification problem more difficult, they also make a solution more valuable.
+
+Several neural network approaches yielded promising results in enhancer prediction.
+Both Basset [@doi:10.1101/gr.200535.115] and DeepEnhancer [@tag:Min2016_deepenhancer] used CNNs to predict enhancers.
+DECRES used a feed-forward neural network [@doi:10.1101/041616] to distinguish between different kinds of regulatory elements, such as active enhancers, and promoters.
+DECRES had difficulty distinguishing between inactive enhancers and promoters.
+They also investigated the power of sequence features to drive classification, finding that beyond CpG islands, few were useful.
+
+Comparing the performance of enhancer prediction methods illustrates the problems in using metrics created with different benchmarking procedures.
+Both the Basset and DeepEnhancer studies include comparisons to a baseline SVM approach, gkmSVM [@doi:10.1371/journal.pcbi.1003711].
+The Basset study reports gkmSVM attains a mean auPRC of 0.322 over 164 cell types [@doi:10.1101/gr.200535.115].
+The DeepEnhancer study reports for gkmSVM a dramatically different auPRC of 0.899 on nine cell types [@tag:Min2016_deepenhancer].
+This large difference means it's impossible to directly compare the performance of Basset and DeepEnhancer based solely on their reported metrics.
+DECRES used a different set of metrics altogether.
+To drive further progress in enhancer identification, we must develop a common and comparable benchmarking procedure (see Discussion).
+
+#### Promoter-enhancer interactions
+
+In addition to the location of enhancers, identifying enhancer-promoter interactions in three-dimensional space will provide critical knowledge for understanding transcriptional regulation.
+SPEID used a CNN to predicted these interactions with only sequence and the location of putative enhancers and promoters along a one-dimensional chromosome [@doi:10.1101/085241].
+It compared well to other methods using a full complement biochemical data from ChIP-seq and other epigenomic methods.
```

Comment:
`complement of biochemical`

Reply:
Fixed in michaelmhoffman/deep-review@af9e92bcfd1464dc2932286e2cffa1ca7f4c2fbb.